### ISSUE TITLE
Hunger Games shouldn't save answers as roboto-app user

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/openfoodfacts-hungergames-react#13]
**Rationale**
- Sometimes I answer too quickly and I need to get back to a particular product: it's easier to find it while being saved by my user account
- I like to know how much products I have done during a Hunger Game session.

(Robotoff might be responsible but I open the ticket where I observ the problem.)

**Comment by teolemon** [comment]
The robotoff API already has a setting to consider the insight applied by the client.
https://github.com/openfoodfacts/robotoff/blob/master/doc/api.md

This would require either:
- creating an auth mechanism on Hunger Games
- having a mechanism to let Robotoff impersonate the user

**Comment by CharlesNepote** [comment]
I don't understand your answer. A few weeks ago, Hunger Games used to save my answers with my user account. Using robotoff user account to save my answers is a regression.

I would put this issue in P1.

**Comment by teolemon** [comment]
``` 
update (int, optional) - Send the update to Openfoodfacts if update=1, don't send the update otherwise. This parameter is useful if the update is performed client-side.
``` 
@ArnaudBarre at some point, Hunger Games was using the cookie to perform the update on its own ?
I can't remember why we changed the behavior ? RobotOFF now has the ability not to perform the update himself.
